docs/_posts/0000-03-01-object-orientation.md

```diff
@@ -25,6 +25,18 @@ Note:
 Alan Turing - who is he?  
 The notion of Turing Completeness and the fact that all the languages below the line on the previous slide have the same programming power.  
 
+--
+
+## Alan Kay and Smalltalk
+
++ [Coined the term](https://wiki.c2.com/?HeInventedTheTerm) ‘Object Orientated Programming’ around 1967
+  + [May not have invented Objects](https://wiki.c2.com/?HeDidntInventTheTerm)
++ Led research at Xerox PARC which resulted in the Smalltalk language
++ [Said this great thing](https://medium.com/javascript-scene/the-forgotten-history-of-oop-88d71b9b2d9f):
```

Comment:
Maybe split the quote into a separate slide, doesn't fit too well on my screen at least.

Reply:
Happy to. It was a little cramped. 👍